integration-tests/tests/spec-utils.js

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2020 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+module.exports = {
+
+  timesMakeSense: async function(browser, eventsArr, startName, endName) {
```

Comment:
maybe halpful: If this function was attached to `browser`, we could save typing thanks to one argument less, maybe something like `browser.timesMakeSense = (...args) => specUtils.timesMakeSense(browser, ...args)`

Reply:
Yes, much cleaner, thanks.